node_content_crash.patch

```diff
@@ -0,0 +1,112 @@
+diff --git a/config/filtering/display_filters.yml b/config/filtering/display_filters.yml
+new file mode 100644
+index 0000000..d5eef68
+--- /dev/null
++++ b/config/filtering/display_filters.yml
+@@ -0,0 +1,22 @@
++allowed_types:
++  #entry per CT, NO default on purpose! It might crash the server if a default value contains wrong items
++  person:
++    - Main
++    - Image
++    - Profile links
++
++  peer_reviewed_article:
++    - Main
++    - Details
++
++  resource:
++    - Main
++    - Details
++
++  front: {}
++  page: {}
++  event: {}
++  news: {}
++  organisation: {}
++  system: {}
++  blog: {}
+diff --git a/src/Plugin/Field/FieldType/ViewfieldItem.php b/src/Plugin/Field/FieldType/ViewfieldItem.php
+index d3a3813..45a2fd1 100644
+--- a/src/Plugin/Field/FieldType/ViewfieldItem.php
++++ b/src/Plugin/Field/FieldType/ViewfieldItem.php
+@@ -2,14 +2,21 @@
+ 
+ namespace Drupal\viewfield\Plugin\Field\FieldType;
+ 
++use Drupal;
+ use Drupal\Core\Entity\ContentEntityStorageInterface;
+ use Drupal\Core\Field\Plugin\Field\FieldType\EntityReferenceItem;
+ use Drupal\Core\Field\FieldStorageDefinitionInterface;
+ use Drupal\Core\Field\FieldDefinitionInterface;
+ use Drupal\Core\Form\FormStateInterface;
++use Drupal\Core\TypedData\ComplexDataDefinitionInterface;
+ use Drupal\Core\TypedData\DataDefinition;
+ use Drupal\Core\Field\FieldFilteredMarkup;
++use Drupal\Core\TypedData\TypedDataInterface;
++use Drupal\node\Entity\Node;
++use Drupal\node\Entity\NodeType;
+ use Drupal\views\Views;
++use Symfony\Component\Yaml\Exception\ParseException;
++use Symfony\Component\Yaml\Yaml;
+ 
+ /**
+  * Plugin implementation of the 'viewfield' field type.
+@@ -26,6 +33,18 @@ use Drupal\views\Views;
+  */
+ class ViewfieldItem extends EntityReferenceItem {
+ 
++  private const YML_PATH = __DIR__ . '/../../../../config/filtering/display_filters.yml';
```

Comment:
should we rather do something like:
`$module_handler = \Drupal::service('module_handler');
$module_path = $module_handler->getModule('THE_NAME')->getPath();`

Reply:
@slejnej Fair suggestion, I've updated the patch